res/openvpn-gui-res-jp.rc

```diff
@@ -452,20 +454,20 @@ Use openvpn-gui --help for more info."
     IDS_ERR_OPEN_SCMGR "OpenSCManagerに失敗しました。 (%d)"
     IDS_ERR_STOP_SERVICE "OpenVPNサービスの停止に失敗しました。"
     IDS_NFO_RESTARTED "OpenVPNサービスを再起動しました。"
-    IDS_ERR_ACCESS_SERVICE_PIPE "Access to service pipe failed."
-    IDS_ERR_WRITE_SERVICE_PIPE "Writing to service pipe failed."
-    IDS_ERR_INSTALL_ISERVICE     """OpenVPNServiceInteractive"" is not installed.\n\
-Tasks requiring administrative access may not work."
-    IDS_ERR_NOTSTARTED_ISERVICE  """OpenVPNServiceInteractive"" is not started.\n\
-Tasks requiring administrative access may not work."
-    IDS_ERR_NOTSTARTED_ISERVICE_ADM  """OpenVPNServiceInteractive"" is not started.\n\
-Wintun driver will not work."
+    IDS_ERR_ACCESS_SERVICE_PIPE "サービスパイプへのアクセスに失敗しました。"
+    IDS_ERR_WRITE_SERVICE_PIPE "サービスパイプへの書き込みに失敗しました。"
+    IDS_ERR_INSTALL_ISERVICE     """OpenVPNServiceInteractive"" がインストールされていません。\n\
+管理者権限で実行すると動作する可能性があります。"
```

Comment:
This wording is suggesting to run as admin when iservice is not installed. I thing we should say the connection may fail without the service. I want to encourage the use of iservice :)
Same below.

Reply:
OK, I understand that the use of this service is strongly recommended.
I'll update this message soon.